home/templates/home/explore.html

```diff
@@ -6,15 +6,23 @@
 <body class="main-bg">
 
 <form method="GET" class="container">
-<div class="m-5" >
+<div class="m-3" >
 
-    <div class="row mb-4">
+    <div class="row my-4">
         <div class="col-2">
             <a href="{% url 'new_question' %}" class="btn shlifim-btn" role="button">Ask new question</a>
         </div>
 
     </div>
 
+    {% if tag != None %}
+    <div class="tags-msg p-3">
+        <p><b>Showing all questions that include the tag: #{{tag}}</b></p>
```

Comment:
maybe space this for readability -> {{ tag }}

Reply:
@liornoy 
Done